rq/queue.py

```diff
@@ -84,7 +89,8 @@ def all(
 
         def to_queue(queue_key):
             return cls.from_queue_key(
-                as_text(queue_key), connection=connection, job_class=job_class, serializer=serializer
+                as_text(queue_key), connection=connection, job_class=job_class,
+                death_penalty_class=death_penalty_class, serializer=serializer
```

Comment:
`death_penalty_class` should be added after `serializer`, otherwise it would break codes using positional arguments.